test2/test2/Controllers/HomeController.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+using test2.Data;
+using test2.Models;
+using Microsoft.EntityFrameworkCore;
+using System.Text.Json;
+
+namespace test2.Controllers
+{
+    public class HomeController : Controller
+    {
+        private readonly CountryContext _context;
+
+        public HomeController(CountryContext context)
+        {
+            _context = context;
+        }
+
+        public IActionResult Index()
+        {
+            return View();
+        }
+        public JsonResult GetCountries(
+            [FromQuery] bool sea_check = false,
+            [FromQuery] long salary_min = 0,
+            [FromQuery] long salary_max = 0,
+            [FromQuery] long area_min = 0,
+            [FromQuery] long area_max = 0,
+            [FromQuery] long people_min = 0, 
+            [FromQuery] long people_max = 0)
+        {
+            List<CountryModel> db = _context.CountryModel.ToList();

```

Comment:
```suggestion
            List<CountryModel> countryModels = _context.CountryModel.ToList();

```
Здесь итак видно что это список объектов, однако ниже по коду используется `db`, что конечно вызывает вопросы